client/src/components/DropdownMenu.js

```diff
@@ -28,50 +30,44 @@ export class DropdownMenu extends React.Component {
     } = this.props;
     const { isOpen } = this.state;
 
+    const aria_label = _.isObject(dropdown_trigger_txt)
```

Comment:
Can rename `dropdown_raw_txt` to dropdown_trigger_a11y_txt, or something like that?

Also this variable should be used on line 83